src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+objects. Now, It looks as if in Hegel’s time scent was thought to be transmitted
+in a non-material way. Obviously, this is no longer the case, but Hegel’s point
+is clear: `communication` is an ideal process that occurs between material
+objects. In other words, it is not just material processes that occur between
+material objects. Here, Hegel is clearly going against a materialist/mechanistic
+worldview whereby the only processes are _material_ processes. Perhaps Hegel is
+thinking of forces and, perhaps more specifically, the force of gravity as an
+example of `communication`.
+
+At the end of this long paragraph where Hegel gives examples of `communication`,
+he expands on an important _logical_ dimension of this moment. He writes:
+
+> The formal totality of the object in general, which is indifferent to the
+> determinateness and hence is not a self-determination, makes it
+> undistinguished from the other object and thus renders the interaction
+> primarily an unimpeded continuation of the determinateness of the one in the
+> other (Hegel 1991, 716) `Communication` does not work simply because the
+> determinateness between `mechanical objects` is identical but also because
+> they are indifferent to determination. Their indifference means that the
+> identical determinateness, in other words, the determinateness that is
+> _communicated_, continues from one object to the other without any alteration
+> in that determinateness. It just continues exactly as it did from the
+> beginning. Thus, not only does `communication` begin from the identity of
+> `mechanical objects` but it also continues with that identity throughout the
+> interaction.
+
+Hegel, then, concludes this section on `communication` with an analysis of how
+it functions in the “spiritual sphere” (Hegel 1991, 716). By spiritual sphere,
+Hegel has in mind the following: “Laws, morals, rational conceptions in general”
+(Hegel 1991, 716). There are laws in a society, and these laws pervade (are
+communicated) throughout society without any obstruction or alteration. The same
+with morals, what a society takes to be right or wrong is immediately
+communicated throughout the society and, again, there is no alteration of
+morality (on a social level) as it goes from individual to individual.[^3]: Of
+course, this is based on an understanding of individuals within a society as
+mechanical objects that are part of mechanical processes. From this paragraph
+alone, it is not clear whether Hegel thinks that laws or morals are examples of
+`communication` or that he thinks that _they can be_ used or expressed in the
+form of `communication`. I suspect that he thinks that the proliferation of
+laws, customs, etc, are logically structured according to `communication`. See:
+Nathan Ross. Let us have a look at the first sentence of this paragraph to get a
+better understanding of what Hegel is going for:
+
+> Now in the spiritual sphere there is an infinitely manifold content that is
```

Comment:
Are you ending the section or the footnote with this quote? Seems a bit of a cliff-hanger to just end with a large quote.

Reply:
Yep, my mistake. Have amended.